openapi/kas-fleet-manager.yaml

```
@@ -970,6 +1014,68 @@ components:
         plan:
           description: kafka plan in a format of <instance_type>.<size_id>
           type: string
+    SupportedKafkaInstanceTypesList:
+      allOf:
+        - type: object
+          example:
+            instance_types:
+              $ref: '#/components/examples/SupportedKafkaInstanceTypeListExample'
+          properties:
+            instance_types:
+              type: array
+              items:
+                allOf:
+                  - $ref: '#/components/schemas/SupportedKafkaInstanceType'
+    SupportedKafkaInstanceType:
+      description: 'Supported Kafka instance type'
+      type: object
+      properties:
+        id:
+          description: 'Unique identifier of the Kafka instance type.'
+          type: string
+        sizes:
+          description: ' A list of Kafka instance sizes available for this instance type'
+          type: array
+          items:
+            allOf:
+              - $ref: '#/components/schemas/SupportedKafkaSize'
+    SupportedKafkaSize:
+      description: 'Supported Kafka Size'
+      type: object
+      properties:
+        id:
+          description: 'Unique identifier of this Kafka instance size.'
+          type: string
+        ingress_throughput_per_sec:
+          description: 'Ingress throughput per second available to this Kafka instance size.'
```

Let's make it clear that this is the maximum limit. 

```suggestion
          description: 'Maximum Ingress throughput per second available to this Kafka instance size.'
```

---

Thanks I'll update those descriptions